WebSocketiny/WebSocketiny.csproj

```diff
@@ -0,0 +1,6 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<LangVersion>8</LangVersion>
```

Comment:
You can optionally enable nullability alalyzers if you want
```xml
<Nullable>enable</Nullable>
```

Reply:
I'll blame you if borked!